pkgdb2client/cli.py

```diff
@@ -876,13 +877,13 @@ def main():
         if not arg.bzurl.endswith('xmlrpc.cgi'):
             arg.bzurl = '%s/xmlrpc.cgi' % arg.bzurl
         print("Querying bugzilla at: %s" % arg.pkgdburl)
-        utils.BZCLIENT.url = arg.bzurl
-        utils.BZCLIENT._sslverify = not arg.insecure
+        pkgdb2client.utils.BZCLIENT.url = arg.bzurl
+        pkgdb2client.utils.BZCLIENT._sslverify = not arg.insecure
 
     if arg.fasurl:
         print("Querying FAS at: %s" % arg.pkgdburl)
-        utils.FASCLIENT.base_url = arg.fasurl
-        utils.FASCLIENT.insecure = arg.insecure
+        pkgdb2client.utils.FASCLIENT.base_url = arg.fasurl
+        pkgdb2client.utils.FASCLIENT.insecure = arg.insecure
```

Comment:
The change is for the cli.py file, that does not import ```as utils```: https://github.com/fedora-infra/packagedb-cli/blob/master/pkgdb2client/cli.py#L28

Also the ```pkgdb2client.``` prefix is also used in other lines already: https://github.com/fedora-infra/packagedb-cli/blob/master/pkgdb2client/cli.py#L435